builtin/logical/aws/path_static_roles.go

```diff
@@ -0,0 +1,271 @@
+package aws
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/fatih/structs"
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+	"github.com/hashicorp/vault/sdk/queue"
+)
+
+const (
+	pathStaticRole = "static-roles"
+
+	paramRoleName       = "name"
+	paramUsername       = "username"
+	paramRotationPeriod = "rotation_period"
+)
+
+type staticRoleConfig struct {
```

Comment:
nit: should we change this to something like `staticRoleEntry`? This would also be more in line with the naming conventions we use in path_roles.go -- [awsRoleEntry](https://github.com/hashicorp/vault/blob/c4294ac515551867fa9b3316dcd5981b5b2a5ece/builtin/logical/aws/path_roles.go#L510).

Reply:
done!